tests/utils/expressions.js

```diff
@@ -0,0 +1,37 @@
+export const getProductNameExpression = ( productTitle ) =>
+	`a[contains(text(), "${ productTitle }")]`;
+
+export const getQtyItemExpression = ( args ) =>
+	'div[@class="wc-block-cart-item__quantity" and div[@class="wc-block-components-quantity-selector" and ' +
+	getQtyInputExpression( args ) +
+	']]';
+
+export const getQtyInputExpression = ( args = {} ) => {
+	let qtyValue = '';
+
+	if ( args.qty ) {
+		qtyValue = ` and @value="${ args.qty }"`;
+	}
+
+	return (
+		'input[contains(@class, "wc-block-components-quantity-selector__input")' +
+		qtyValue +
+		']'
+	);
+};
+
+export const getQtyPlusButtonExpression = () => {
+	return 'button[contains(@class, "wc-block-components-quantity-selector__button--plus")]';
+};
+
+export const getQtyMinusButtonExpression = () => {
+	return 'button[contains(@class, "wc-block-components-quantity-selector__button--minus")]';
+};
+
+export const getCartItemExpression = ( productTitle, args ) =>
+	'//tr[contains(@class, "wc-block-cart-items__row") and ' +
```

Comment:
I would suggest changing the name of any of these expression functions to include XPath instead (for example `getCartItemExpression` -> `getCartItemXPath`, to give it more context as to what the syntax is. (I'd never seen it before for example, and I thought you were just commenting out that line!)

Reply:
`XPath` is an expression language. And the text returned by each of the above functions is called `expression`. So, it's more accurate to name the functions by their return type instead of the language itself. But, we may change it to `PathExpression` instead to keep an accurate naming and help anyone who isn't familiar with `XPath` to follow the code (e.g., `getCartItemPathExpression `).

Reply:
- Renamed functions by adding: `PathExpression` at the end.
- Renamed the expression file: `path-expressions`